public_html/formatter.js

```diff
@@ -233,28 +233,41 @@ function format_latlng(p) {
 }
 
 function format_data_source(source) {
-	switch (source) {
-		case 'mlat':
-			return "MLAT";
-		case 'adsb_icao':
-		case 'adsb_other':
-			return "ADS-B";
-		case 'adsb_icao_nt':
-			return "ADS-B (non transponder)";
-		case 'adsr_icao':
-		case 'adsr_other':
-			return "ADS-R";
-		case 'tisb_icao':
-		case 'tisb_trackfile':
-		case 'tisb_other':
-			return "TIS-B";
-		case 'mode_s':
-			return "Mode S";
-		case 'mode_ac':
-			return "Mode A/C";
+	let sourceString = "";
+
+	if (source.has('uat')) {
+		sourceString += "UAT ";
 	}
 
-	return "";
+	if ((source.has('adsb_icao') || source.has('adsb_other')) && !source.has('uat')) {
```

Comment:
Could we create variables for these and then reference the variables in the `if` blocks? It's nitpicky but it may be a little cleaner.

```suggestion
      let adsb_ico = source.has('adsb_icao');
      let adsb_other = source.has('adsb_other');
      let uat = source.has('uat');

	if ((adsb_icao || adsb_other) && !uat) {
```

Reply:
I like it